<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QWidget" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库查询</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>查询类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="t_year">
         <property name="text">
          <string>年报表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="t_mon">
         <property name="text">
          <string>月报表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="t_day">
         <property name="text">
          <string>日报表</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>268</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="s_year"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>年</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="s_mon"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>月</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="s_day"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>日</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="t_select">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>138</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>438</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="t_close">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
